### PR TITLE
update of configuration

### PR DESCRIPTION
**Pull request by sjaakd**

### CODE REVIEW

File: mapstruct-lombok/pom.xml [hunk 1]
```diff
@@ -36,22 +36,18 @@
...
 
         <dependency>
             <groupId>org.mapstruct</groupId>
             <artifactId>mapstruct</artifactId>
             <version>${org.mapstruct.version}</version>
         </dependency>
+
+        <!-- lombok dependencies should not end up on classpath -->
         <dependency>
-            <groupId>org.mapstruct</groupId>
```
Comment (filiphr) [inline mapstruct-lombok/pom.xml]: If you remove the mapstruct processor then stuff won't work in IntelliJ pre 2018.1.1

Comment (sjaakd) [reply mapstruct-lombok/pom.xml]: @filiphr  Is this still an issue? Should we leave it as (than I'll close this one)?

Comment (filiphr) [reply mapstruct-lombok/pom.xml]: I think we are fine now. IntelliJ is releasing 2019.1 in few weeks and I think people update regularly.

**Comment by sjaakd** [comment]
@filiphr thanks..

**Pull request merged**